recipes/default.rb

```diff
@@ -19,8 +19,10 @@
 
 influxdb_config node['influxdb']['config_file_path'] do
   config node['influxdb']['config']
+  notifies :restart, 'service[influxdb]', :delayed
 end
 
 service 'influxdb' do
-  action [:enable, :start]
+  supports :restart => true
```

Comment:
I think Ruby will interpret that as a block. But you can use `supports(restart: true)`